staging/src/k8s.io/kubectl/pkg/cmd/rollout/rollout_status.go

```diff
@@ -228,4 +230,10 @@ func (o *RolloutStatusOptions) Run() error {
 			return err
 		})
 	})
+
+	if !resourceFound {
```

Comment:
If err is not nil, we are still showing this warning message. I think, we can show this warning only when err is nil. What do you think?.

I mean would it be better something like;

```go
if err != nil {
    return err
}
if !resourceFound {
....
```

Reply:
Yes. I think that's correct so we don't show a double message. Thanks!